saidump/saidump.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <inttypes.h>
+
+// Default value: 100MB
+constexpr int64_t RDB_JSON_MAX_SIZE = 1024 * 1024 * 100;
+
+struct CmdOptions
+{
+    bool skipAttributes;
+    bool dumpTempView;
+    bool dumpGraph;
+    std::string rdbJsonFile;
+    uint64_t rdbJSonSizeLimit;
+};
+
+void printUsage();
+CmdOptions handleCmdLine(int argc, char **argv);
```

Comment:
Do not make global functions make them as static in class look for syncd commandline

Reply:
Hi @kcudnik, could you please give more details about your comments?  If I modified them as below:
```
static void printUsage();
staic CmdOptions handleCmdLine(int argc, char **argv);
```
When I compiling unittest/saidump, there were errors below:
```
jumao@8f4d6eacacca:/sonic/src/sonic-sairedis/unittest/saidump$ make check
g++ -DHAVE_CONFIG_H -I. -I../..    -g -I../../SAI/inc -I../../SAI/experimental -I../../SAI/meta -I../../saidump -I../../lib -I../../vslib -I../../meta -ansi -fPIC -std=c++14 -Wall -Wcast-align -Wcast-qual -Wconversion -Wdisabled-optimization -Werror -Wextra -Wfloat-equal -Wformat=2 -Wformat-nonliteral -Wformat-security -Wformat-y2k -Wimport -Winit-self -Wno-inline -Winvalid-pch -Wmissing-field-initializers -Wmissing-format-attribute -Wmissing-include-dirs -Wmissing-noreturn -Wno-aggregate-return -Wno-padded -Wno-switch-enum -Wno-unused-parameter -Wpacked -Wpointer-arith -Wredundant-decls -Wshadow -Wstack-protector -Wstrict-aliasing=3 -Wswitch -Wswitch-default -Wunreachable-code -Wunused -Wvariadic-macros -Wwrite-strings -Wno-switch-default -Wconversion -Wno-psabi -Wcast-align=strict -g -O2 -MT tests-main.o -MD -MP -MF .deps/tests-main.Tpo -c -o tests-main.o `test -f 'main.cpp' || echo './'`main.cpp
In file included from main.cpp:3:
../../saidump/saidump.h:16:13: error: 'void printUsage()' declared 'static' but never defined [-Werror=unused-function]
   16 | static void printUsage();
      |             ^~~~~~~~~~
../../saidump/saidump.h:16:13: error: 'void printUsage()' used but never defined [-Werror]
cc1plus: all warnings being treated as errors
make: *** [Makefile:496: tests-main.o] Error 1

```

Did you mean to add a saidump class to include all the functions? Such as below:

```
class saidump
{
    static void printUsage();
    staic CmdOptions handleCmdLine(int argc, char **argv);
}

```

Thank you.